beacon-chain/sync/pending_attestations_queue.go

```diff
@@ -143,10 +143,12 @@ func (s *Service) processPendingAtts(ctx context.Context) error {
 				}
 			}
 
-			req := [][]byte{bRoot[:]}
+			req := [][32]byte{bRoot}
 			if err := s.sendRecentBeaconBlocksRequest(ctx, req, pid); err != nil {
-				traceutil.AnnotateError(span, err)
-				log.Errorf("Could not send recent block request: %v", err)
+				if err = s.sendRecentBeaconBlocksRequestFallback(ctx, req, pid); err != nil {
```

Comment:
Again, i think we should whitelist this with `altona` flag

Reply:
This is not really for altona, it is for our current Onyx nodes so that we do not break backwards compatibility